src/XR/features/WebXRControllerPointerSelection.ts

```diff
@@ -724,6 +882,27 @@ export class WebXRControllerPointerSelection extends WebXRAbstractFeature {
         // here due to a typo
         return this.laserPointerDefaultColor;
     }
+
+    private _pickWithMesh(indexTipMesh: AbstractMesh, precise: boolean, predicate: (mesh: AbstractMesh) => boolean): Nullable<PickingInfo> {
+        var pickingInfo = new PickingInfo();

```

Comment:
May as well not do this PickingInfo allocation unless we know we can use it. Since you're returning a Nullable object, you can just return null when the pick fails, instead of returning a default picking info. Either that or don't make the return nullable if there's no way to return null.